pkg/sdkserver/localsdk.go

```diff
@@ -161,11 +165,20 @@ func (l *LocalSDKServer) SetExpectedSequence(sequence []string) {
 	l.expectedSequence = sequence
 }
 
+// SetSdkName set SDK name to be added to the logs
+func (l *LocalSDKServer) SetSdkName(sdkName string) {
+	l.testSdkName = sdkName
+	l.logger = l.logger.WithField("sdkName", l.testSdkName)
```

Comment:
Perfect. Love it.

Reply:
Thanks now at least logs from local SDK server would be labeled with SDK name. Hope to label SDK test clients in the future (second half of logs).